api/src/services/streams-service.ts

```diff
@@ -47,9 +48,11 @@ export class StreamsService {
 		pskId: string;
 	}> {
 		try {
+			console.log("Making seed", seed)
```

Comment:
I don't think we should have these logs. Otherwise our code would speak a lot.

Reply:
Cleanup done of forgotten statements